dashboard/report/style.css

```
@@ -0,0 +1,25 @@
+body {
+    font-family: Arial, sans-serif;
```

I recommend setting `font-family` to a standard windows font stack as the default/Arial fonts do not look to visually appealing

i.e.
```suggestion
      font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
```